### PR TITLE
Fixing a few compiler complaints in PPCValidatorClient.

### PR DESCRIPTION
**Pull request by thirtified**
A few fixes to make master compile again.

### CODE REVIEW

File: PayPalCommercePlatform/PPCValidatorClient.m [hunk 1]
```diff
@@ -34,7 +34,7 @@ - (nullable instancetype)initWithAccessToken:(NSString *)accessToken {
         NSError *error;
         _payPalUAT = [[BTPayPalUAT alloc] initWithUATString:accessToken error:&error];
         if (error || !_payPalUAT) {
-            NSLog(@"[PayPalCommercePlatformSDK]: Error initializing PayPal UAT. Error code: %@", error.code);
+            NSLog(@"[PayPalCommercePlatformSDK]: Error initializing PayPal UAT. Error code: %ld", (long) error.code);
```
Comment (scannillo) [inline PayPalCommercePlatform/PPCValidatorClient.m:37]:
```suggestion
            NSLog(@"[PayPalCommercePlatformSDK]: Error initializing PayPal UAT. Error code: %i", error.code);
```

**Comment by scannillo** [comment]
Hey @thirtified. Pulling the latest master, I'm able to compile just fine without these changes. Can you show an example of the compilation error you see / your compiler settings?

Regardless - these are some good changes to make!

File: PayPalCommercePlatform/PPCValidatorClient.m (hunk 1 above)
Comment (thirtified) [reply PayPalCommercePlatform/PPCValidatorClient.m:37]: NSError's `code` property is an NSInteger which depending on the platform will have a length of 32 or 64 bits and should therefore be cast to a long for display as an integer. Given the available error code range I guess there won't ever be a problem – but it seems good practice, wouldn't you agree?

**Comment by thirtified** [comment]
> Pulling the latest master, I'm able to compile just fine without these changes. Can you show an example of the compilation error you see / your compiler settings?

Hm, interesting. When you explicitly compile PPCValidatorClient.m you should see the errors. You will also see the errors if you remove `inhibit_all_warnings!` from the Podfile.

Do you think it may be worth to get together and validate the current project structure? I seem to be running into issues from time to time (e.g. currently can't get the PayPalCommercePlatform target to build at all) and also have at least one idea for improvements (use configuration files for customisable build parameters such as team name, Apple Pay Merchant IDs etc.).

**Comment by scannillo** [comment]
Thanks for submitting these fixes!

I spoke with our EM, Gary Bramwell, and he asked if you'd be able to give us a write up of your suggestions for SDK improvements/design? This way our entire team can check out your ideas and then kick off the conversation from there?

Also - happy to chat in Slack more.

**Pull request merged**